debian/ubuntu-advantage-tools.postinst

```diff
@@ -355,6 +355,15 @@ remove_old_systemd_timers() {
     fi
 }
 
+remove_old_config_fields() {
+    PREVIOUS_PKG_VER="$1"
+    if dpkg --compare-versions "$PREVIOUS_PKG_VER" le "27.8"; then
+        if grep -q license_check_log_file /etc/ubuntu-advantage/uaclient.conf; then
```

Comment:
@orndorffgrant no idea where the inline comment went. TLDR was:
1. Maybe we can make `grep` a bit stricter: `grep -q '^license_check_log_file:'`.
2. I checked what mysql-8.0 does (https://git.launchpad.net/ubuntu/+source/mysql-8.0/tree/debian/mysql-server-8.0.postinst?id=e417d5e8ba04504a15491283ba64d82496f60542#n118) and it's similar, which is reassuring. I don't know why they do the `cmp` + `mv` thing, I think what we have here is fine.